ui/src/common/components/__tests__/__snapshots__/LabelWithHelp.test.jsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`LabelWithHelp renders label with help 1`] = `
 <span>
   Label
-   
+   
```

Comment:
what is this?

Reply:
The previous one is an `&nbsp;` and the new one is a normal whitespace.